README.rst

```diff
@@ -220,17 +219,18 @@ Contributors
 `zrax <//github.com/zrax>`_
 
 
-.. |pypi| image:: http://img.shields.io/pypi/v/urwid.svg
+.. |pypi| image:: https://img.shields.io/pypi/v/urwid
     :alt: current version on PyPi
     :target: https://pypi.python.org/pypi/urwid
 
-.. |docs| image:: https://readthedocs.org/projects/urwid/badge/
-    :alt: docs link
-    :target: http://urwid.readthedocs.org/en/latest/
+.. |docs| image:: https://readthedocs.org/projects/urwid/badge/?version=latest
```

Comment:
readthedocs changed domain (redirect `http://urwid.readthedocs.org/en/latest/` -> `https://urwid.readthedocs.io/en/latest` is executed automatically, but not wait for old domain close)